pyproject.toml

```diff
@@ -19,6 +19,7 @@ dev = [
     "pytest-xdist",
     "pytest>=8",  # Pin to keep recent
     "python-dotenv",
+    "qdrant-client",
```

Comment:
No you need to do `paper-qa[datasets,ldp,qdrant,typing,zotero,local]` on line 9, don't duplicate the package

Reply:
Great save. Thanks.